.github/workflows/sphinx-make-page.yml

```diff
@@ -24,12 +24,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install --upgrade --requirement requirements.txt
-    - name: Install generator parameter library
+    - name: Install generate_parameter_library
       run: |
         cd
         git clone https://github.com/PickNikRobotics/generate_parameter_library.git
         cd generate_parameter_library/generate_parameter_library_py/
-        sudo python setup.py install
+        pip install pyyaml
```

Comment:
The way of calling pip should be consistent. You either call `pip ...` or `python -m pip` unless you want different pips by not using the default python but it seems that's the case here

Reply:
this is only a temporary fix until https://github.com/PickNikRobotics/generate_parameter_library/pull/123 gets merged. But I'll update it